build/Dockerfile.multiarch

```diff
@@ -0,0 +1,10 @@
+FROM golang:buster AS builder
+
+COPY . /build
+WORKDIR /build
+RUN go build -o ./_output/bin/piraeus-operator -gcflags all=-trimpath=. --asmflags all=-trimpath=. github.com/piraeusdatastore/piraeus-operator/cmd/manager
```

Comment:
I suspect the issue with distroless/static is that the go build command doesn't produce a static binary. You might want to use `CGO_ENABLED=0 go build --ldflags '-extldflags "-static"' ...` and try again.

Reply:
Yes, I saw it was a dynamic binary and short googling did not produce a command that would generate a static binary. `CGO_ENABLED` looks promising, though. Now I'm learning go and github actions. I just wanted storage :-D

Reply:
Yes, now it's static.